strutils/reverse_test.go

```diff
@@ -29,6 +29,24 @@ func TestReverseBytes(t *testing.T) {
 	}
 }
 
+func TestCoverage(t *testing.T) {
```

Comment:
Can you solve this problem like Test_checkUTF8Len

Reply:
ok，i'll solve, how about like this `Test_ReverseBytesEmpty`

Reply:
I mean coverage is done by looping test cases, not by individual test methods

Reply:
sorry !  is my fault